ts/packages/node/src/stores/Keyv.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable spellcheck/spell-checker */
+import { KeyValueConfigStore } from '@configu/ts';
+import type Keyv from 'keyv';
+
+export type KeyvConfigStoreConfiguration = { keyvInstance: Keyv };
+
+/**
+ * KeyvConfigStore is a KeyValueConfigStore implementation that uses Keyv as a backend.
+ * @description
+ * Keyv is a simple key-value store for Node.js with support for multiple backends.
+ * @see https://www.npmjs.com/package/keyv
+ * @example
+ * ```ts
+ * import Keyv from 'keyv';
+ * import { KeyvConfigStore } from '@configu/ts-node';
+ * const keyvMongo = new Keyv('mongodb://user:pass@localhost:27017/dbname');
+ * const configStore = new KeyvConfigStore({ keyvInstance: keyvMongo });
+ * ```
+ */
+export class KeyvConfigStore extends KeyValueConfigStore {
+  client: Keyv;
+
+  constructor({ keyvInstance }: KeyvConfigStoreConfiguration) {
```

Comment:
if we wish to make this store available in the cli, we should somehow allow passing the Keyv constructor parameter type (e.g the mongo connection string / other store connection options) as the `KeyvConfigStoreConfiguration` so it can be supplied as strings and booleans  and call `new Keyv` in the constructor.

Reply:
That's a great idea! 

We could write our constructor based on the Keyv constructor parameter (e.g the mongo connection string / other options), so that way we would instantiate the keyv instance

but for the mongo connection string to work, the client would have to install the mongo adapter alongside the Configu package. So we would need to make it clear on the docs.

To use KeyvConfigStore with mongodb, It could be something similar to this:
### Install
```bash
npm install @configu/node @keyv/mongo
```
### Use
```ts
const configStore = new KeyvConfigStore('mongodb://user:pass@localhost:27017/dbname')
```
in the example above Keyv will automatically load the correct storage adapter, as the client installed @keyv/mongo alongside @configu/node